nf_core/modules/lint/main_nf.py

```diff
@@ -290,8 +290,9 @@ def check_process_section(self, lines, fix_version, progress_bar):
             else:
                 self.failed.append(("docker_tag", "Unable to parse docker tag", self.main_nf))
                 docker_tag = None
-            if l.startswith("biocontainers/"):
-                # When we think it is a biocontainer, assume we are querying quay.io/biocontainers and insert quay.io as prefix
+            # Guess if container name is simple one (e.g. nfcore/ubuntu:20.04)
+            # If so, add quay.io as default container prefix
+            if l.count("/") == 1 and l.count(":") == 1:
```

Comment:
will this handle containers listed just as `ubuntu:20.04` ?
if not maybe as below? 
```suggestion
            if l.count("/") <= 1 and l.count(":") == 1:
```

Reply:
ah my bad! didn't pull the latest changes for modules...i see all are in `nf-core/ubuntu:20.04` format...lgtm